packages/ui-proxy/src/create-proxy-server.js

```diff
@@ -95,6 +95,11 @@ function createProxyServer({
         console.log( // eslint-disable-line no-console
             `Check ${proxyAddress}/endpoint/application/run/genestack/signin`
         );
+    }).on('error', (error) => {
+        if (error.code === 'EADDRINUSE') {
+            console.error('\x1b[31m', `Port ${proxyPort} is in use, retry with another port`);
```

Comment:
Could you please move `'\x1b[31m'` to a constant (e.g. `COLOR_RED`). And the same with `COLOR_YELLOW`.

Reply:
And the better message could be: 
```
`Port ${proxyPort} is in use, retry with another port using --server=<port>`
```